src/components/BalanceInput/BalanceInput.tsx

```diff
@@ -1,14 +1,13 @@
 import React from 'react'
 import styled from 'styled-components'
-import { Button, Box, BoxProps, Flex, Text } from '@pancakeswap-libs/uikit'
+import { Button, Box, BoxProps, Flex, Input, Text } from '@pancakeswap-libs/uikit'
 import useI18n from 'hooks/useI18n'
-import Input from './Input'
 
-interface BalanceInputProps extends BoxProps {
+export interface BalanceInputProps extends BoxProps {
   title: string
   max: number
   symbol: string
-  onChange: (evt: React.FormEvent<HTMLInputElement>) => void
+  onChange?: (evt: React.FormEvent<HTMLInputElement>) => void
```

Comment:
What is the reason why `onChange` is optional?

Reply:
No, sorry
Fixed